src/ORM/RulesChecker.php

```diff
@@ -102,12 +108,9 @@ public function existsIn(
         Table|Association|string $table,
         array|string|null $message = null
     ): RuleInvoker {
-        $options = [];
-        if (is_array($message)) {
-            $options = $message + ['message' => null];
-            $message = $options['message'];
-            unset($options['message']);
-        }
+        $options = is_array($message) ? $message : ['message' => $message];
+        $message = $options['message'] ?? null;
+        unset($options['message']);
```

Comment:
While certainly shorter by two lines, is this really considered more readable than the previous style?